modules/bootkube/resources/bootkube.sh

```diff
@@ -25,15 +25,15 @@ echo "Rendering MCO manifests..."
 		--dest-dir=/assets/mco-bootstrap \
 		--images-json-configmap=/assets/manifests/machine-config-operator-01-images-configmap.yaml
 
-mkdir -p /etc/kubernetes/manifests/
-mkdir -p /etc/mcc/bootstrap/
-mkdir -p /etc/ssl/mcs/
+mkdir --parents /etc/kubernetes/manifests/
+mkdir --parents /etc/mcc/bootstrap/
+mkdir --parents /etc/ssl/mcs/
 
 # Bootstrap MachineConfigController uses /etc/mcc/bootstrap/manifests/ dir to
 # 1. read the controller config rendered by MachineConfigOperator
 # 2. read the default MachineConfigPools rendered by MachineConfigOperator
 # 3. read any additional MachineConfigs that are needed for the default MachineConfigPools.
-cp -r "$PWD/mco-bootstrap/manifests" /etc/mcc/bootstrap/manifests
+cp --recursive "$PWD/mco-bootstrap/manifests" /etc/mcc/bootstrap/manifests
```

Comment:
News to me, but [POSIX.1-2017 dropped `-r` in favor of `-R`][1] and does not define a long form.  They give some notes on `-r` vs. `-R` in the RATIONALE section.

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/cp.html